private_acl_rules.tf

```diff
@@ -54,6 +54,23 @@ resource "aws_network_acl_rule" "private_ingress_from_operations_mattermost_web"
   from_port      = 8065
   to_port        = 8065
 }
+# Allow ingress from first private subnet via UDP ephemeral ports.
```

Comment:
Is this allowing ingress from the first private subnet _to_ the private subnets?  I'm confused as to why this would be necessary.

Reply:
You are correct- I had this rule somewhat backwards before.  I have corrected it, improved the comment (hopefully), and tested it to my satisfaction with https://github.com/cisagov/cool-assessment-terraform/pull/206/commits/34051abde3b014b5e82eda1945769a0d7edabbb3.  Let me know what you think.